packages/amplify_authenticator/lib/src/services/amplify_auth_service.dart

```diff
@@ -18,7 +18,6 @@ import 'dart:async';
 import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
 import 'package:amplify_flutter/amplify_flutter.dart';
 import 'package:collection/collection.dart';
-import 'package:collection/src/iterable_extensions.dart';
```

Comment:
Removing this was needed to get the authenticator to a max pub score

Reply:
I'm assuming it was superfluous and we had access to the extensions via `collection.dart`?

Reply:
Correct